### DISCUSSION TITLE
How to have more than 2 extra apps

### DISCUSSION THREAD

**Discussion by MathisMillot** [SideStore/SideStore#294] [Q&A] [answered]
Is there a way to have more than 2 extra apps using sidestore ?

**Comment by amsam0** [comment, marked as answer]
Yes, using MDC. There is a beta version that has it built into SideStore, you can join our discord to help test. (It only works on iOS 16.1.2 and lower)

**Reply by MathisMillot** [reply]
your discord invite is expired do u have a valid link

**Reply by eon-pixel** [reply]
> your discord invite is expired do u have a valid link

The one found on their website [https://sidestore.io](url) is valid, unless you're banned.
